spec/type_aliases/ip_address_v6_nosubnet_full_spec.rb

```diff
@@ -1,28 +1,24 @@
 require 'spec_helper'
 
-if Puppet::Util::Package.versioncmp(Puppet.version, '4.5.0') >= 0
-  describe 'Stdlib::IP::Address::V6::Nosubnet::Full' do
```

Comment:
I think this line should not have been removed

Reply:
added back